<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDFDialog</class>
 <widget class="QDialog" name="PDFDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adding new PDF set</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resource.qrc">
    <normaloff>:/icons/logo.png</normaloff>:/icons/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>PDF Error:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>PDF Set:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboPDFset">
     <item>
      <property name="text">
       <string>ToyLH (APFEL internal)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="4" rowspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="comboPDFerror">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>NNPDF (Monte Carlo)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>LHAPDF</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QDial" name="dialDLAGP">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>1</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>APFEL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>APFEL Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Theory:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_theory">
        <item>
         <property name="text">
          <string>QCD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QED</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QECDP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCEDP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QavDP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QECDS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QCEDS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QavDS</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2" rowspan="7">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Symbol';&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; vertical-align:sub;&quot;&gt;S &lt;/span&gt;(Q&lt;span style=&quot; vertical-align:sub;&quot;&gt;ref&lt;/span&gt;) =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="lineEdit_as">
        <property name="text">
         <string>0.350</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Pert. Order:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_ptord">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>LO (LO QED)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NLO (LO QED)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NNLO (LO QED)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Q&lt;span style=&quot; vertical-align:sub;&quot;&gt;ref&lt;/span&gt; (GeV) =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="lineEdit_Qas">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>1.4142135623731</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Scheme:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_scheme">
        <item>
         <property name="text">
          <string>VFNS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FFNS, 3 flavors</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FFNS, 4 flavors</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FFNS, 5 flavors</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FFNS, 6 flavors</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Symbol';&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; vertical-align:sub;&quot;/&gt;(Q&lt;span style=&quot; vertical-align:sub;&quot;&gt;ref&lt;/span&gt;) =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEdit_a">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.007496252</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>HQ scheme:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_hqscheme">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>POLE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MSbar</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Q&lt;span style=&quot; vertical-align:sub;&quot;&gt;ref&lt;/span&gt; (GeV) =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="lineEdit_Qa">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>1.777</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;m&lt;span style=&quot; vertical-align:sub;&quot;&gt;c &lt;/span&gt; (GeV) =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_mc">
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="inputMask">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>1.4142135623732</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Symbol';&quot;&gt;m&lt;/span&gt;&lt;span style=&quot; vertical-align:sub;&quot;&gt;R &lt;/span&gt;/&lt;span style=&quot; vertical-align:sub;&quot;/&gt;&lt;span style=&quot; font-family:'Symbol';&quot;&gt;m&lt;/span&gt;&lt;span style=&quot; vertical-align:sub;&quot;&gt;F&lt;/span&gt; =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_murmuf">
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;m&lt;span style=&quot; vertical-align:sub;&quot;&gt;b&lt;/span&gt; (GeV) =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_mb">
        <property name="text">
         <string>4.5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Max. Flavor PDFs:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QSpinBox" name="spinBox_maxpdf">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;m&lt;span style=&quot; vertical-align:sub;&quot;&gt;t&lt;/span&gt; (GeV) =&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_mt">
        <property name="text">
         <string>175.0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Max. Flavor &lt;span style=&quot; font-family:'Symbol';&quot;&gt;a:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QSpinBox" name="spinBox_maxa">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>lineEdit_Qas</zorder>
     <zorder>comboBox_scheme</zorder>
     <zorder>line_3</zorder>
     <zorder>lineEdit_as</zorder>
     <zorder>label_5</zorder>
     <zorder>label_17</zorder>
     <zorder>label_6</zorder>
     <zorder>comboBox_ptord</zorder>
     <zorder>comboBox_theory</zorder>
     <zorder>label_8</zorder>
     <zorder>label_16</zorder>
     <zorder>label_18</zorder>
     <zorder>lineEdit_a</zorder>
     <zorder>comboBox_hqscheme</zorder>
     <zorder>label_7</zorder>
     <zorder>label_19</zorder>
     <zorder>lineEdit_Qa</zorder>
     <zorder>lineEdit_mc</zorder>
     <zorder>doubleSpinBox_murmuf</zorder>
     <zorder>label_20</zorder>
     <zorder>label_23</zorder>
     <zorder>lineEdit_mb</zorder>
     <zorder>spinBox_maxpdf</zorder>
     <zorder>label_24</zorder>
     <zorder>label_21</zorder>
     <zorder>spinBox_maxa</zorder>
     <zorder>lineEdit_mt</zorder>
     <zorder>label_25</zorder>
     <zorder>label_22</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PDFDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PDFDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
